src/pretix/base/email.py

```diff
@@ -469,6 +470,19 @@ def base_placeholders(sender, **kwargs):
                 }
             ),
         ),
+        SimpleFunctionalMailTextPlaceholder(
+            'last_order_modification_date_and_time', ['order', 'event'],
```

Comment:
IMHO the name is a bit misleading as I initially read it as the last time the order was modified – I think either `latest_order_modification_date_and_time` or maybe even better something with _deadline_:
```suggestion
            'order_modification_deadline_date_and_time', ['order', 'event'],
```